src/logic/operate.js

```diff
@@ -20,5 +20,19 @@ export default function operate(numberOne, numberTwo, operation) {
       return one.div(two).toString();
     }
   }
+  if (operation === "**2") {
+    if(one === 0){
+      if(two > 1){
+        if (two % 2 === 0){
+          if(two % 3 === 0){
+            return "way too complex"
+          }
+        }
+      }
```

Comment:
Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/gh/dan-codacy/calculator/pullRequest?prid=6273880)